coos.html

```diff
@@ -460,43 +464,51 @@ <h3>Base GSBPM model</h3>
-
-				<pre class="example" title="Example of statistical production activity">
-				ex:produce-study a coos:StatisticalProductionActivity ;
-					rdfs:label "Produce study"@en .
-				</pre>
-
-				<p><code>Phase</code> and <code>SubProcess</code> are both sub-classes of <code>StatisticalProductionActivity</code> and represent the concepts of a GSBPM phase and GSBPM sub-process.</p>
-
-				<pre class="example nohighlight" title="Examples of GSBPM phase and sub-process">
-				&lt;http://id.unece.org/activities/phase/6&gt; a coos:Phase ;
-					skos:notation "6" ;
-					skos:prefLabel "Analyse"@en .
-
-				&lt;http://id.unece.org/activities/subProcess/7.2&gt; a coos:SubProcess ;
-					skos:notation "7.2" ;
-					skos:prefLabel "Produce dissemination products"@en .
-				</pre>
-
-				<p>The ontology defines an individual for each GSBPM phase and sub-process, as well as two specific individuals: the <code>GSBPM</code> itself, which is viewed as as a <code>skos:ConceptScheme</code> (the scheme of all the GSBPM phases and sub-processes), and <code>statisticalProductionProcess</code>, which is an instance of <code>StatisticalProductionActivity</code> that represents the statistical production process as a whole.</p>
-
-				<p>The links between the different individuals are represented by the usual SKOS properties: all the sub-processes of a phase have a <code>skos:broader</code> link to the phase, and each phase has a <code>skos:broader</code> link to the <code>statisticalProductionProcess</code> individual. Additionally, the idea that each phase is made up of its different sub-processes can be represented by using the Dublin Core <code>dcterms:hasPart</code> and <code>dcterms:isPartOf</code> properties.</p>
+				<p>A first approach to OWL modelling of statistical activities based on the GSBPM is presented in [[GSBPM-LM]].</p>
+				
+				<p>To render the GSBPM as a global taxonomy of statistical activities, the ontology defines a class <code>ActivityCategory</code> that is a sub-class of <code>skos:Concept</code>. For each GSBPM phase and sub-process an invidual is then defined, as well as two specific individuals: the <code>GSBPM</code> itself, which is viewed as a <code>skos:ConceptScheme</code> (the scheme of all the GSBPM phases and sub-processes), and <code>statisticalProductionProcess</code>, which is an instance of <code>ActivityCategory</code> that represents the statistical production process as a whole.</p>
 
+				<p>The links between the different individuals are represented by the usual SKOS properties: all the sub-processes of a phase have a <code>skos:broader</code> link to the phase, and each phase has a <code>skos:broader</code> link to the <code>statisticalProductionProcess</code> individual.</p>
```

Comment:
... which somehow imply that <code>statisticalProductionProcess</code> belongs to the GSBPM scheme, correct ?

Reply:
Yes